ns2-stat-api/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "ns2-stat-api"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4.0.1"
+clap = { version = "3.1.6", default_features = false, features = ["std", "derive"] }
+ns2-stat = { path = "../ns2-stat" }
+serde = "1.0.136"
+serde_json = "1.0.79"
```

```suggestion
actix-web = "4.0"
clap = { version = "3.1", default_features = false, features = ["std", "derive"] }
ns2-stat = { path = "../ns2-stat" }
serde = "1.0"
serde_json = "1.0"
```
I prefer onl< specifying `<major>.<minor>`, since that allows using newer (compatible) patches.